cmd/recommend.go

```diff
@@ -34,6 +34,7 @@ func init() {
 
 	recommendCmd.Flags().StringSliceVar(&recommendOptions.Images, "image", []string{}, "Container image list (comma separated)")
 	recommendCmd.Flags().StringSliceVar(&recommendOptions.Uselabels, "use-labels", []string{}, "User defined labels for policy (comma separated)")
+	recommendCmd.Flags().StringVarP(&recommendOptions.Usenamespace, "use-namespace", "", "", "User defined namespace value for policies")
```

Comment:
We can have shorthand options here.
- `-i` for `--image`
- `-l` for `--use-labels`
- `-n` for `--use-namespace`

Reply:
Hey @wazir-ahmed , use-labels and use-namespace should not use the same shorthand imo ... We will need those shorthands subsequently for k8s based recommendations. CC: @vishnusomank

Reply:
@nyrahul What do you think about this.. 
- We will remove `use-labels` and `use-namespace` options. And add `labels` and `namespace` options.
- If user invokes the `recommend` command with `--image`, then we consider it as container-image based recommendation.
```
karmor recommend -i ubuntu:20.04 -l app=ubuntu -n test
```
If the user invokes the command without `--image` option, then we consider it as k8s based recommendation.  
```
karmor recommend -l app=feeder -n test
```

Reply:
That's a good proposition. @vishnusomank, WDYT?

Reply:
Sounds good. This will also make the user experience better. With some minor adjustments, we can achieve this too

Reply:
@nyrahul 
I will add the above mentioned changes as part of k8s manifest changes. The PR is in good shape. We can merge this.